src/main/java/org/jenkinsci/test/acceptance/po/Jenkins.java

```diff
@@ -112,7 +113,14 @@ public void restart() {
         visit("restart");
         clickButton("Yes");
 
-        waitFor(driver, not(hasContent("Please wait")), JenkinsController.STARTUP_TIMEOUT);
+        try {
+            waitFor(driver, not(hasContent("Please wait")), JenkinsController.STARTUP_TIMEOUT);
+        }catch(TimeoutException e) {
+            //Let's try to avoid false negatives or not auto refresh
+            visit(driver.getCurrentUrl());
+            //we wait 10 seconds for refresh things.
+            waitFor(driver, hasContent("New Item"), 10);
```

Comment:
if you have a secured instance (not sure if ATHs have this on restart?) then you may not see the "New Item" part.